source/includes/_wallet.md

```diff
@@ -790,6 +823,10 @@ const account = 'default';
 
 Add a shared xpubkey to wallet. Must be a multisig wallet.
 
+<aside class="notice">
+Note that since it must be a multisig, the wallet on creation should be set with <code>m</code> and <code>x</code> where <code>n</code> is greater than 1 (since the first key is always that wallet's own xpubkey)
```

Comment:
`<code>m</code> and <code>x</code>` should be `<code>m</code> and <code>n</code>`